docker/domserver/Dockerfile

```diff
@@ -10,12 +10,8 @@ RUN apt-get update \
 	gcc g++ make acl zip unzip \
 	php-cli php-zip \
 	php-gd php-curl php-mysql php-json php-intl \
-	php-gmp php-xml php-mbstring \
 	sudo bsdmainutils ntp libcgroup-dev procps \
-	linuxdoc-tools linuxdoc-tools-text groff \
 	python3-sphinx python3-sphinx-rtd-theme python3-pip fontconfig python3-yaml \
-	texlive-latex-recommended texlive-latex-extra \
```

Comment:
Don’t we need latex to build some docs still?

Reply:
I checked in the sources and for both `master` and `7.3` in the install texlive is not needed (but it is for `7.2`).

According to @thijskh the last `pdflatex` in the Makefiles is not needed, I checked locally and also the `groff` seems to not be needed to build the docs for master.